src/app/js/main.js

```diff
@@ -191,7 +191,25 @@ function submit_jira(slide) {
   slide.appendChild(element)
 }
 
-
+// call our server rest api 
+function submitReport(data) {
+  const button = document.getElementById('jiraBtn');
+  button.addEventListener('click', function (e) {
+    console.log('button was clicked');
```

Comment:
make this specific for the submit report (i.e. subitReport button was clicked) or remove the log statement.

Reply:
I will cleanup these debugging statements. thanks